Yoast/wordpress-seo
# Remove -ms- prefixes everywhere

This remove all of the automatic `-ms-` prefixes everywhere. Does this by switching to new version of plugin-grunt-tasks, so as to switch to new browserslist implementation.

## Summary


This PR can be summarized in the following changelog entry:

* Remove IE11 specific CSS.

## Test instructions

This PR can be tested by following these steps:

* `yarn install`.
* `grunt build:css`: no -ms- prefixes should be found in the built css.
* Click through the plugin for 2 minutes.

## Documentation

## Quality assurance


Fixes #
